tests/sch/rotated-resistor.test.tsx

```diff
@@ -20,4 +20,4 @@ test("schematic rotated resistor", () => {
   expect(
     convertCircuitJsonToSchematicSvg(circuit.getCircuitJson() as any),
   ).toMatchSvgSnapshot(import.meta.path)
-})
+}, 20000)
```

Comment:
put `routingDisabled`/`pcbDisabled` in the board. We should solve the underlying issue here at some point